composer.json

```diff
@@ -24,13 +24,13 @@
         "sylius/api-bundle": "^2.0",
         "sylius/core-bundle": "^2.0",
         "sylius/resource-bundle": "^1.12",
-        "symfony/config": "6.4.*",
-        "symfony/dependency-injection": "6.4.*",
-        "symfony/form": "6.4.*",
-        "symfony/http-foundation": "6.4.*",
-        "symfony/http-kernel": "6.4.*",
-        "symfony/routing": "6.4.*",
-        "symfony/uid": "6.4.*",
+        "symfony/config": "6.4.* || ^7.0",
```

Comment:
lets go directly to 7.3 
```suggestion
        "symfony/config": "6.4.* || ^7.3",
```
<img width="901" alt="image" src="https://github.com/user-attachments/assets/a5d6e214-631d-4c0d-9a82-3dcae34ddb52" />